lib/nimble_template/addons/test_env.ex

```diff
@@ -133,10 +133,7 @@ defmodule NimbleTemplate.Addons.TestEnv do
   end
 
   defp edit_test_support_cases(project) do
-    project
-    |> edit_test_support_case("channel_case")
-    |> edit_test_support_case("conn_case")
-    |> edit_test_support_case("data_case")
+    edit_test_support_case(project, "data_case")
```

Comment:
I guess we still have the `conn_case` on the Phoenix 1.6.11 🤔should we keep the `edit_test_support_cases(conn_case)` there @byhbt

Reply:
The main purpose of `edit_test_support_case` here is about clean the alias `Ecto.Adapters.SQL.Sandbox`. But on the new version, the `alias` is not used anymore. So I think we can remove the `conn_case` in this case.

They reference directly to the `datacase`:

```
setup tags do            
    SampleApp.DataCase.setup_sandbox(tags)            
                    
    {:ok, conn: Phoenix.ConnTest.build_conn()}            
end
```

Reference: https://www.phoenixdiff.org/?source=1.6.6&target=1.6.11 `test/support/conn_case.ex`

Reply:
Got it @byhbt